data/battle_anim_scripts.s

```diff
@@ -24500,3 +24575,5632 @@ Special_CriticalCaptureBallThrow:
+	playsewithpan SE_M_MEGA_KICK, SOUND_PAN_ATTACKER
+	call ContinentalCrushStockpileRocks
+	playsewithpan SE_M_MEGA_KICK, SOUND_PAN_ATTACKER
+	call ContinentalCrushStockpileRocks
+	launchtemplate gContinentalCrushGrowingRockSpriteTemplate 0x2 0x1 0x0
+	launchtask AnimTask_BlendColorCycle 0x2 0x6 0x2 0x2 0x2 0x0 0xb 0x2dd3
+	launchtask AnimTask_ShakeMon2 0x2 0x5 0x0 0x1 0x0 0x20 0x1
+	playsewithpan SE_M_MEGA_KICK, SOUND_PAN_ATTACKER
+	call ContinentalCrushStockpileRocks
+	playsewithpan SE_M_MEGA_KICK, SOUND_PAN_ATTACKER
+	call ContinentalCrushStockpileRocks
+	invisible ANIM_ATTACKER
+	waitforvisualfinish
+	fadetobg BG_IN_AIR
+	waitbgfadeout
+	visible ANIM_TARGET
+	launchtask AnimTask_StartSlidingBg 0x5 0x4 0x0 0xf800 0x0 0xffff
+	waitbgfadein
+	loopsewithpan SE_FALL, SOUND_PAN_ATTACKER, 0x20, 0x4 @;Falling sound
+	delay 0x1e
+	launchtemplate gContinentalCrushEruptionSpriteTemplate 0x28 0x5 0xc8 0xffe0 0x0 0x64 0x0
+	launchtemplate gContinentalCrushEruptionSpriteTemplate 0x28 0x5 0x1e 0xffe0 0x10 0x5a 0x1
+	launchtemplate gContinentalCrushEruptionSpriteTemplate 0x28 0x5 0x96 0xffe0 0x20 0x3c 0x2
+	launchtemplate gContinentalCrushEruptionSpriteTemplate 0x28 0x5 0x5a 0xffe0 0x30 0x50 0x3
+	launchtemplate gContinentalCrushEruptionSpriteTemplate 0x28 0x5 0x6e 0xffe0 0x40 0x32 0x0
+	launchtemplate gContinentalCrushEruptionSpriteTemplate 0x28 0x5 0x3c 0xffe0 0x50 0x46 0x1
+	delay 0x16
+	launchtask AnimTask_HorizontalShake 0x5 0x3 0x5 0x8 0x1c
+	launchtask AnimTask_HorizontalShake 0x5 0x3 0x4 0x8 0x1c
+	unloadspritegfx ANIM_TAG_ROUND_SHADOW	@ fly
+	unloadspritegfx ANIM_TAG_MUD_SAND @ dig
+	unloadspritegfx ANIM_TAG_FOCUS_ENERGY @ focus energy
+	unloadspritegfx ANIM_TAG_IMPACT @ hit
+	loadspritegfx ANIM_TAG_REALLY_BIG_ROCK
+	delay 0x30
+	launchtemplate gContinentalCrushBigRockStompSpriteTemplate, 0x82, 0x4, 0, 0x3c, 3, ANIM_TARGET
+	launchtask AnimTask_BlendBattleAnimPal 0xa 0x5 0x4 0x2 0x0 0x7 0x0000
+	delay 0x5
+	playsewithpan SE_M_EXPLOSION, SOUND_PAN_TARGET
+	launchtemplate gSlideMonToOffsetSpriteTemplate 0x2 0x5 ANIM_TARGET 0xff00 0x15 0x0 0x4
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x0 0xc 0xffe8 0xfff0 0x18 	@ 12, -48, -16, 24
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x0 0x10 0xffda 0xfff6 0x18 	@ 16, -16, -10, 24
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x1 0xe 0xffec 0xffee 0x18 	@ 14, -52, -18, 24
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x1 0xc 0xffdc 0xfff0 0x18 	@ 12, -32, -16, 24
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x0 0xc 0xffd0 0xfff0 0x18 		@ 12, -24, -16, 24
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x0 0x10 0xfff0 0xfff6 0x18 		@ 16, -38, -10, 24
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x1 0xe 0xffcc 0xffee 0x18 		@ 14, -20, -18, 24
+	launchtemplate gDirtPlumeSpriteTemplate 0x2 0x6 ANIM_TARGET 0x1 0xc 0xffe0 0xfff0 0x18 		@ 12, -36, -16, 24
```

Comment:
The comments should all be aligned when possible, imo. There's no reason for these 4 not to be aligned with the 4 above them.